### PR TITLE
{lib}[GCCcore/10.3.0] M1QN3 v3.3

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3110.skitty.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/40243fcf1b77c78b1310b91007e85dd2 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3516.doduo.os - Linux RHEL 8.4, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/3980f368004ed991771aa1a79075dd57 for a full test report.

File: easybuild/easyconfigs/m/M1QN3/M1QN3-3.3-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+easyblock = 'CmdCp'
+
+name = 'M1QN3'
+version = '3.3'
+
+homepage = 'https://who.rocq.inria.fr/Jean-Charles.Gilbert/modulopt/optimization-routines/m1qn3/m1qn3.html'
+description = "A solver of large-scale unconstrained minimization problems"
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
```
Comment (branfosj) [inline easybuild/easyconfigs/m/M1QN3/M1QN3-3.3-GCCcore-10.3.0.eb:9]: Is this something that we would also want to install with intel compilers?

Comment (boegel) [reply easybuild/easyconfigs/m/M1QN3/M1QN3-3.3-GCCcore-10.3.0.eb:9]: Good point, it's Fortran, so yes...

I'll move this to `GCC` toolchain

Comment (boegel) [reply easybuild/easyconfigs/m/M1QN3/M1QN3-3.3-GCCcore-10.3.0.eb]: fixed in caf2012

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node2673.swalot.os - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/f9113f1956b77255c9a0a2fb9c6d8867 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.5, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/6ce7f302ce284e6f67e02280cf3c33df for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**